CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 ## [2.1.0] - 2022-08-17
-### Added
```

Comment:
Is this intentional

Reply:
It's not, first time editing a changelog and I didn't notice that header, sorry.